source/reference/limitations.txt

```diff
@@ -64,6 +64,11 @@ General Limitations
 - You can't sync a collection with any documents that have an empty timestamp,
   such as ``Timestamp(0,0)`` on pre-6.0 source clusters. 
 
+.. important::
+
+   ``mongosync`` enforces the documented limitations, regardless of whether
```

Comment:
Succinctness:

```
``mongosync`` enforces the limitations...
```

The "documented limitations" implies undocumented limitations.

Reply:
Ah that makes sense. Thank you for this fix!